tests/_testerlib.livecodescript

```diff
@@ -41,13 +41,30 @@ function TesterGetTestFileNames pBaseFolder
       put the defaultfolder into pBaseFolder
    end if
 
-   TesterGetTestFileNames_Recursive pBaseFolder, empty, tFiles, tCount
+   TesterGetActualFileNames_Recursive pBaseFolder, empty, tFiles, tCount
 
    return tFiles
 end TesterGetTestFileNames
 
+-- Get all compiler test files beneath the CWD, apart from
+-- filenames starting with "." or "_"
+function TesterGetCompilerTestFileNames pBaseFolder
+   local tFiles, tCount
+
+   put empty into tFiles
+   put 0 into tCount
+
+   if pBaseFolder is empty then
+      put the defaultfolder into pBaseFolder
+   end if
+
+   TesterGetActualFileNames_Recursive pBaseFolder, empty, "compilertest", tFiles, tCount
+
+   return tFiles
+end TesterGetCompilerTestFileNames
```

Comment:
I suggest adding an optional `pExtension` argument to `TesterGetTestFileNames()` instead of adding this function (which is highly duplicative).